Dynatrace/dynatrace-operator
# Adds dynakube namespace to list of ignored namespaces

# Description

Re-adds a dynakube's namespace to its list of namespaces that are ignored by default.

## How can this be tested?

* Deploy the operator
* Deploy a dynakube with application monitoring and activegate (or deploy sample apps in dynakube's namespace)
* Make sure nothing is injected


## Checklist
